src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -206,7 +206,10 @@ private SyntaxNode GenerateForwardedTypeAssemblyAttributes(IAssemblySymbol assem
                 }
                 else
                 {
-                    _logger.LogWarning(string.Format("Could not find matching assembly: '{0}' in any of the search directories.", $"{symbol.ContainingAssembly.Name}.dll"));
+                    _logger.LogWarning(string.Format(
+                        "Could not resolve type '{0}' in containing assembly '{1}' via type forward. Make sure that the assembly is provided as a reference and contains the type.",
+                        symbol.ToString(),
```

Comment:
nit: Should this be `ToDisplayString()`?

Reply:
updated